content/hi/docs/reference/glossary/network-policy.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Network Policy
```

Comment:
I'd expect to see “NetworkPolicy” here; the words “Network” and “Policy” aren't Hindi.

Reply:
You can use `aka:` to also provide the original name.